plugins/source/aws/resources/services/dynamodb/backups.go

```diff
@@ -0,0 +1,71 @@
+package dynamodb
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func Backups() *schema.Table {
+	tableName := "aws_dynamodb_backups"
+	return &schema.Table{
+		Name:                tableName,
+		Description:         `https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BackupDescription.html`,
+		Resolver:            listBackups,
+		PreResourceResolver: getBackup,
+		Multiplex:           client.ServiceAccountRegionMultiplexer(tableName, "dynamodb"),
+		Transform:           transformers.TransformWithStruct(&types.BackupDescription{}),
+		Columns: []schema.Column{
+			client.DefaultAccountIDColumn(false),
+			client.DefaultRegionColumn(false),
+			{
+				Name:     "arn",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("BackupDetails.BackupArn"),
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
```

Comment:
I wonder, do we want to start labeling the PK columns `NotNull` explicitly, or do a transition at some point?
cc: @yevgenypats

Reply:
To set NotNull explicitly for all PKs

Reply:
Why wouldn't we just do it in the SDK?